htdocs/mri/jiv/get_file.php

```diff
@@ -57,9 +57,11 @@
     exit(2);
 }
 
-// Now get the file and do file validation
-$File = $_GET['file'];
+// Now get the file and do file validation.
+// Resolve the filename before doing anything.
+$File = realpath($_GET['file']);
```

Comment:
@driusan @johnsaigle 

As I checkout the branch with these changes, I no longer see my images, nor can I download the minc images from imaging browser.
As I print `$File` to the error_log, in the first case, I get the relative path of the file printed out, in the second, I get nothing. @johnsaigle, did this change work for you?